src/extension.node.ts

```diff
@@ -19,7 +19,13 @@ require('./platform/logging');
 // locations at which we record various Intervals are marked below in
 // the same way as this.
 
-const durations: Record<string, number> = {};
+const durations = {
+    totalActivateTime: 0,
+    codeLoadingTime: 0,
+    startActivateTime: 0,
+    endActivateTime: 0,
+    workspaceFolderCount: 0
```

Comment:
We were missing the necessary type annotations, now telemetry is strongly typed and we know exactly what is sent